frontend/src/components/interaction-manager/content/log.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react'
+
+import { Intent } from '../../../config/intent'
+import { AppToaster } from '../../toaster'
+import type { InteractionContentProps, Log } from '../types'
+
+const getMessageIntent = (type: string) => {
+  switch (type) {
```

Comment:
chore (non-blocking): I think a map would be better here, less code branches.

```js
const INTENT_MAP = {
  error: Intent.DANGER,
  ...
}

// Usage
INTENT_MAP[data.type] || Intent.NONE
```

Reply:
used to love the maps, but for some reason switch statements grew on me lately. any reason why the preference for object lookups? i think I'd need the same amount of object props as switch cases. also, switch is almost twice as fast as lookups (in modern browsers):

<img width="750" alt="Screenshot 2022-10-13 at 10 50 25" src="https://user-images.githubusercontent.com/105208209/195550895-b0a91b98-e454-416a-b1c0-5c8cdc224fd5.png">

Reply:
More code branches from a switch means higher cyclomatic complexity which i prefer to keep as low as possible. I don't really mind much about performance for something like this personally. I'm not massively fussed either way hence non blocking. I'll leave it up to you 👍